clustering/ejb/WFLY-16345_SFSB_externalization_to_remote_Infinispan_server.adoc

```diff
@@ -0,0 +1,91 @@
+= @Stateful Jakarta Enterprise Bean instance externalization to remote Infinispan cluster
+:author:            Paul Ferraro
+:email:             paul.ferraro@redhat.com
+:toc:               left
+:icons:             font
+:idprefix:
+:idseparator:       -
+
+== Overview
+
+Currently, WildFly only supports the use of embedded Infinispan for persisting/replicating @Stateful Jakarta Enterprise Bean instances.
+This implementation cannot fully support cross-site replication, since the inability to invalidate entries across sites means that site failback can result in loss of state.
```

Comment:
@pferraro it's not clear to me how, with the implementation proposed in this RFE, we can overcome this limitation

Reply:
An invalidation cache, by definition caches entries.  If this invalidation cache uses a shared remote Infinispan cluster as a cache store, and any entries are update in a remote site, there is no mechanism to invalidate the entries within the invalidation cache.